### PR TITLE
Add newsroom section on landingPage

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add newsroom section on landingPage:
  - update gatsby backend for blog view: add getter for latest blog posts
  - update landingPage view

**Related issue(s)**
Part of #363

### CODE REVIEW

File: src/views/landingPage/SocialMedia/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { LastBlogPosts } from "./LastBlogPosts";
+import { YouTubeFrame } from "./YouTubeFrame";
+import { TwitterFrame } from "./TwitterFrame";
+
+import { SocialMediaI18nTarget } from "@typings/landingPage";
+import { Post } from "@typings/blog";
+
+import { SocialMediaWrapper } from "./styled";
+
+import config from "../../../../config.json";
```
[the remark this answers is not in the document]
Comment (aerfio) [reply src/views/landingPage/SocialMedia/index.tsx]: or "@config", I don't remember how it was named

File: src/views/landingPage/SocialMedia/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+export const Card: React.FunctionComponent<Props> = ({
+  i18nTarget,
+  iconName,
+  iconPrefix,
+  link,
+  children,
+}) => {
+  const LinkType = link.startsWith("http")
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply src/views/landingPage/SocialMedia/Card.tsx]: Ok, so I'll name import with than name :)

File: src/layouts/default/components/Footer.tsx [hunk 3]
```diff
@@ -31,7 +31,6 @@ const Footer: React.FunctionComponent = () => {
     resolveSocialMedia("slack"),
     resolveSocialMedia("linkedIn"),
     resolveSocialMedia("youTube"),
-    resolveSocialMedia("stackOverflow"),
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply src/layouts/default/components/Footer.tsx]: As I remember, we decide to remove stackoverflow.

File: src/views/landingPage/SocialMedia/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply src/views/landingPage/SocialMedia/index.tsx]: import config from "@config" :) Thanks!

**Comment by magicmatatjahu** [comment]
@aerfio Thanks for review.  

```
I think there's too much white space under check your videos, don't you think? Same with mobile view
```

I know that, but this PR will be merged after PR with used by section and then I will adjust margins :)

**Comment by aerfio** [comment]
@magicmatatjahu ok then 👍

**Pull request merged**